frontend/src/components/organisms/MatchesList/MatchItemType.tsx

```diff
@@ -0,0 +1,21 @@
+import {MatchStatus} from "../../atoms/MatchStatus";
+
+export type MatchItemType = {
```

Comment:
Za duzo tych undefined :) To nam psuje zalety TSa. Tylko tam gdzie rzeczywiscie moze sie tak zdarzyc powinny byc.

Reply:
Niestety tu mogą się zdarzyć :) Np. mecz niewypełniony!